docs/contributing.rst

```diff
@@ -122,8 +122,8 @@ new branch onto your fork, you'll be able to `create a pull request`_.
 Reviewing existing Pull Requests
 --------------------------------
 
-Another very useful way to contribute to COSIMA Recipes is to review existing
-Pull Requests. Even if you don't have a new workflow to propose to the world,
+Another very useful way to contribute to COSIMA Recipes is to review `existing
+Pull Requests`. Even if you don't have a new workflow to propose to the world,
```

Comment:
```suggestion
Pull Requests`_. Even if you don't have a new workflow to propose to the world,
```

Needs an underscore to be a link, I think?

Reply:
Probably! I am not so much into rst; I probably was writing with markdown in mind